odilo.py

```diff
@@ -425,7 +419,8 @@ def record_info_to_metadata(cls, book, availability):
         author = book.get('author')
         if author:
             roles = [Contributor.AUTHOR_ROLE]
-            contributor = ContributorData(sort_name=author, display_name=author, roles=roles, biography=None)
+            sort_author = display_name_to_sort_name(author)
+            contributor = ContributorData(sort_name=sort_author, display_name=author, roles=roles, biography=None)
```

Comment:
This is all right for now but in the long run we should rely on the name canonicalizer from the metadata wrangler to do this work. I think I've mentioned this before.

Reply:
Yes, you told me about it, thanks.